casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -136,6 +144,29 @@ class RuntimeManagerImpl[F[_]: Concurrent] private[rholang] (
     runtime.blockTime.setParams(timestamp)
   }
 
+  private def setInvalidBlocks(
+      invalidBlocks: Map[BlockHash, Validator],
+      runtime: Runtime[F]
+  ): F[Unit] = {
+    val invalidBlocksPar: Par =
+      Par(
+        exprs = Seq(
+          Expr(
+            Expr.ExprInstance.EMapBody(
+              ParMap(SortedParMap(invalidBlocks.map {
+                case (validator, blockHash) =>
+                  (byteStringToPar(validator), byteStringToPar(blockHash))
+              }))
+            )
+          )
+        )
+      )
```

Comment:
isn't this a case for a new RhoType.Map? You could create one with only the apply function and leave the unapply for when we have the first usage

Reply:
We can create it if we need to build a second instance

Reply:
I don't agree. We have rhotype objects for the other types so, for consistency reasons we should have one for map too. It's trivial to do so I don't think it's worth postponing